e2e/test/scenarios/dashboard-filters/reproductions/24235-exlude-all-date-options.cy.spec.js

```diff
@@ -50,7 +50,7 @@ describe("issue 24235", () => {
 
     cy.wait("@getCardQuery");
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Rows 1-10 of 200").should("be.visible");
+    cy.findByText("Rows 1-11 of 200").should("be.visible");
```

Comment:
This starts failing after merging https://github.com/metabase/metabase/pull/31019. Not that PR's fault, but introducing a new banner seems to make the dashboard card size a bit different. I haven't gone into detail yet but here is the result

1. Without database prompt banner (436px)
![Screenshot 2023-06-13 at 1 27 48 PM](https://github.com/metabase/metabase/assets/1937582/e9b8bf99-de9e-4075-a62b-b27a161bfefe)

2. With database prompt banner (446px)
![Screenshot 2023-06-13 at 1 27 02 PM](https://github.com/metabase/metabase/assets/1937582/d7c64da4-8ce0-4d54-a982-0a010dc1f9ff)

The height difference cause the card to contain one more record. I think it's will take too much time to go into detail why this happens and the effect seems very minimal so modifying the change here looks like the best approach for the time being.